src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+// @use "../utils/variables" as *;
+
+.card {
+  box-sizing: border-box;
```

Comment:
Move to * selector and remove from all other places.

Reply:
AD. 1 - You could use universal selector to set `box-sizing: border-box` like so:
```css
*,
*:before,
*:after {
  box-sizing: border-box;
}
```

And you will no longer need to set it to a single element